packages/klighd-core/src/views-rendering.tsx

```diff
@@ -527,7 +527,7 @@ export function renderKText(rendering: KText,
 export function renderChildRenderings(parentRendering: KContainerRendering, parentElement: SKGraphElement, propagatedStyles: KStyles,
     context: SKGraphModelRenderer, childOfNodeTitle?: boolean): (VNode | undefined)[] {
     // children only should be rendered if the parentElement is not a shadow
-    if (!(parentElement instanceof SKNode) || !parentElement.shadow) {
+    if (parentRendering.children && (!(parentElement instanceof SKNode) || !parentElement.shadow)) { // FIXME: is checking parentRendering.children actually necessary?
```

Comment:
is it?

Reply:
I'll find out once I've finished the synthesis-related stuff